addons/easy_charts/Utilities/Scripts/ScatterChartBase.gd

```diff
@@ -0,0 +1,335 @@
+tool
```

Comment:
Rename `ScatterChartBase.gd` to `scatter_chart_base.gd` according to GDScript style guide